Makefile

```diff
@@ -0,0 +1,24 @@
+# File: Makefile
```

Comment:
How about defining a constant to hold acroread, and having that point to evince by default (imho, acroread support was dropped > 2 years ago)?

What about using mklatex to crate the pdf?

Reply:
I see no reference to neither `acroread` nor `evince`. Could you explain what they should be used for?

`mklatex` seems like a good choice :+1: